src/sass/components/footer.sass

```diff
@@ -112,5 +111,54 @@
       border-width: 6px 6px 0 6px
       border-color: $sub-text transparent transparent transparent
 
+// Language switcher
+.lang-switch-link
+  display: inline-block
+  position: relative
+  transition: all 0s ease
+  svg
+    display: inline-block
+    fill: $sub-text
+    margin-bottom: .15em
```

Comment:
In this theme we are using rem instead of em. Lets stick to it.
You could also use rem() mixin